examples/requestor-showcase/pom.xml

```diff
@@ -79,6 +90,29 @@
         </configuration>
       </plugin>
 
+      <!-- USE IT TO ENABLE ANNOTATION PROCESSING IN MAVEN WITHOUT CONFLICTING WITH GWT COMPILATION -->
+      <!--

```

Comment:
Agree. It should be placed in some doc. I let it here because I still didn't start documentating this new version.